exporters/postgres/sql_servers_example.yml

```
@@ -29,4 +29,3 @@ jobs:
           # sql server (eg "postgres-rds-prod", "postgres-rds-dev1" etc.)
           #     'postgres://USERNAME:PASSWORD@DB_SERVER_IP/DATABASE:PORT
           "db1": 'postgres://cht_couch2pg:cht_couch2pg_password@172.17.0.1:5432/cht?sslmode=disable' # //NOSONAR - password is safe to commit
```

Is this ever going to be `cht` database? I think, for the near term, it's always going to be `data` yeah? I got this error initially:

```
ts=2024-10-01T22:28:33.507Z caller=klog.go:134 level=error func=Errorf msg="Error gathering metrics: [from Gatherer #1] [job=db_targets,target=db1] pq: database \"cht\" does not exist"
```

So I think this is the fix. Also, since we default to `postgres` as the user, I think we should use that here too, yeah? Finally - I think we can further demote `couch2pg` by just using `password`:

```suggestion
          "db1": 'postgres://postgres:password@172.17.0.1:5432/code?sslmode=disable' # //NOSONAR - password is safe to commit
```